lib/ruleHelper.js

```diff
@@ -256,6 +269,22 @@ RuleHelper.prototype = {
         case "ArrowFunctionExpression":
             methodName = "";
             break;
+        case "AssignmentExpression":
+            if (RIGHT_OPERATORS.includes(node.operator)) {
+                methodName = this.normalizeMethodCall(node.right);
+            } else if (MATH_OPERATORS.includes(node.operator)) {
+                methodName = "";
+                break;
+            } else if (LOGICAL_OPERATOS.includes(node.operator)) {
+                // Issue #210: oh, no! we have two methods names we need to check, but this function
+                // may only return one. it's not immediately clear how to reconcile that with existing callers of normalizeMethodCall.
+                methodName = "";
```

Comment:
> // Issue #210: oh, no! we have two methods names we need to check, but this function
> may only return one. it's not immediately clear how to reconcile that with existing callers of normalizeMethodCall.

how about changing this method to return an array of method names and on the caller side to expect an array to be always returned and iterate on that array to check all method names returned?